### DISCUSSION TITLE
Host additional files on Security Onion server

### DISCUSSION THREAD

**Discussion by aizande** [Security-Onion-Solutions/securityonion#8052] [Unsupported Versions] [answered]
Hey friends,
Any suggestions on how to have the security onion website host additional files? I was planning on making some exercises and have the players simply download them from the site, the ability to add it into the already hosted document repo or other locations would be a huge plus!

Any tips?

Thanks

**Comment by dougburks** [comment, marked as answer]
You might be able to modify the web server config to do this, but it's not really supported.

**Comment by aizande** [comment]
Thanks doug!
Thats sort of what I figured based off poking around. Thanks for the confirmation :)
